crates/rover-client/src/query/schema/get.rs

```diff
@@ -0,0 +1,108 @@
+use crate::blocking::Client;
+use crate::RoverClientError;
+use graphql_client::*;
+
+// I'm not sure where this should live long-term
+/// this is because of the custom GraphQLDocument scalar in the schema
```

Comment:
probably wouldn't hurt to just put this at the root of the module. do any of our functions return this type? if so, we should definitely expose it as part of our public API so Rover can consume it properly. if not you can just put it at the top level and `pub(crate)` it

Reply:
Right now, I think this is the only place where this type will be used. If we add another query that fetches a schema document, that could change. Do you think it's fine to leave here until it's actually needed multiple places?

Reply:
yeah! totally fine